remove experimental announcement from cloud audit (#9811)
pingcap/docs: tidb-cloud/tidb-cloud-auditing.md

# Database Audit Logging

> > **Note**

Reviewer: ```suggestion
> **Note：**
```

>

> > Database Audit Logging has not been opened for all regions. Please log a support ticket if you don't have the setting in console.

Reviewer: ```suggestion
> The database audit logging feature has not been opened for all regions. If you want to use this feature but do not see **Audit Settings** in the TiDB Cloud console, you can contact the [TiDB Cloud support team](/tidb-cloud/tidb-cloud-support.md) for help.
```

> > The database audit logging feature has not been opened for all regions. If you want to use this feature but do not see **Audit Settings** in the TiDB Cloud console, you can contact the [TiDB Cloud support team](/tidb-cloud/tidb-cloud-support.md) for help.

Reviewer: ```suggestion
> The database audit logging feature is disabled by default. If you want to enable this feature, contact the [TiDB Cloud support team](/tidb-cloud/[tidb-cloud-support.md](http://tidb-cloud-support.md/)) for help.
```
